### ISSUE TITLE
metadata view

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#582]
Consider a view that displays metadata for a selected item.

Requires: https://github.com/cylc/cylc-ui/issues/534
Complicated by: https://github.com/cylc/cylc-ui/issues/434

If a task is selected in the tree view there would be a mechanism (e.g. double click/tap or drag&drop) to share that selection with other views e.g. this "metadata view".

The metadata view would just display all of the Cylc `[meta]` information present for the selected "thing" (i.e. task/family/workflow).

We could potentially support markup languages here (e.g. markdown, rst):

```
[meta]
    title = "My Workflow"
    description = """md
        **Hello** `world`!
    """
    URL = http://whatever.com
```

For example this metadata could be presented in a metadata view like so:

----

### My Workflow

> **id:** myworkflow//

**Hello** `world`!

[http://whatever.com](http://whatever.com)

----

**Pull requests welcome!**

**Comment by oliver-sanders** [comment]
Follow on issues:

* https://github.com/cylc/cylc-ui/issues/1898
* https://github.com/cylc/cylc-ui/issues/1897
* https://github.com/cylc/cylc-ui/issues/1896
* https://github.com/cylc/cylc-ui/issues/1895
* https://github.com/cylc/cylc-ui/issues/1894
* https://github.com/cylc/cylc-ui/issues/1893

**Comment by MetRonnie** [comment]
Duplicate of #1071

**Comment by oliver-sanders** [comment]
> Duplicate of https://github.com/cylc/cylc-ui/issues/1071

Not quite, this issue covers workflow metadata whereas #1071 is specifically about task metadata, hence that issue is closed by #1886 whereas this one isn't.

**Comment by oliver-sanders** [comment]
Issue https://github.com/cylc/cylc-ui/issues/1898 picks up the workflow metadata part.
